backend/pkg/kubeconfig/kubeconfig.go

```diff
@@ -236,10 +238,12 @@ func LoadContextsFromAPIConfig(config *api.Config) ([]Context, []error) {
 			AuthInfo:    authInfo,
 		}
 
-		err := context.SetupProxy()
-		if err != nil {
-			errors = append(errors, fmt.Errorf("couldnt setup proxy for context: %q, err:%q", contextName, err))
-			continue
+		if !skipProxySetup {
```

Comment:
Is there a test for this?

Reply:
It doesn't need specific testing. The function that calls this `LoadContextsFromAPIConfig` has tests already.